cypress/integration/FileInputFieldWithList/Deduplicates_the_file_list/index.js

```diff
@@ -0,0 +1,41 @@
+import '../common'
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+Given('the list contains the file duplicate.md', () => {
+    cy.contains('duplicate.md').then($label => {
+        expect($label).to.have.lengthOf(1)
+        expect($label).to.have.class('label')
+        expect(
+            $label.closest('[data-test="dhis2-uicore-filelistitem"]')
+        ).to.have.lengthOf(1)
+    })
+})
+
+When('the file duplicate.md is selected', () => {
+    cy.get('[data-test="dhis2-uicore-fileinput"] input').then($input => {
+        // name, lastModified, size and type are equal to the file created in the story
+        const duplicate = new File([], 'duplicate.md', {
+            lastModified: 1581348175357,
+            type: 'text/markdown',
+        })
```

Comment:
Duplicating these values can be tricky. If the test falsy succeeds when `lastModified` is accidentally different, it'd be very hard to spot.

Alternatively you can wrap and alias the previously added file with `cy.wrap(...).alias`, access it hiere and use the same `lastModified` value.

I haven't tried async/await in cypress yet, but I'd assume it to work, then it'd be a bit more readable than having nested `cy.get`.

Reply:
I've attached sth to the window from the story as suggested in https://github.com/dhis2/ui-core/pull/787#discussion_r377589620